adapters/facebook.go

```diff
@@ -156,32 +177,50 @@ func (a *FacebookAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder
 		// BANNER
 		fbReqB, err := a.MakeOpenRtbBidRequest(req, bidder, params.PlacementId, pbs.MEDIA_TYPE_BANNER, pubId, i)
 		if err == nil {
-			err = json.NewEncoder(&requests[reqIndex]).Encode(fbReqB)
+			requests[reqIndex] = fbReqB
```

Comment:
Careful here... I think this makes a full copy of the openrtb object, which is pretty huge

If you make `requests` a `[]*openrtb.BidRequest`, that copy doesn't happen.

See also: https://groups.google.com/forum/#!topic/golang-nuts/__BPVgK8LN0